packages/web/src/components/TeeForm.vue

```diff
@@ -226,6 +250,10 @@ const analytics = analyticsStore()
 // const usedTracks: UsedTrack[] | any[] = tracks.getAllUsedTracks
 const trackValues: any[] = tracks.getAllUsedTracksValues
 
+// @ts-ignore
+const metaEnv = import.meta.env
+const contactEmail = ref<string>(metaEnv.VITE_BREVO_SENDER_EMAIL || 'france-transition@beta.gouv.fr')
```

Comment:
suggestion: Why we need a `ref` on `contactEmail`, it seems this don't need to be a dynamic attribute ?

Reply:
Note : I think at the time I needed to display this into the `debug` spaces into the html template. You're right it doesn't need to be reactive

Chore: (for me) I should remove the `ref<string>` occurence

Reply:
note: `ref<string>` use changed in [this commit](https://github.com/betagouv/transition-ecologique-entreprises-widget/pull/135/commits/3df4c62f34668a1ec06d8c70fd3b19ed8acf664e)

Reply:
note: as I changed the target branch and re-synchronised locally my branch `j-form-error-msg` with `preprod`, this change is now included. I can mark this conversation as resolved